basic-application/basic-application-webapp/src/main/java/fr/openwide/core/basicapp/web/application/BasicApplicationApplication.java

```diff
@@ -147,6 +148,9 @@ protected void mountApplicationPages() {
 		mountPage("/administration/user-group/", AdministrationUserGroupPortfolioPage.class);
 		mountParameterizedPage("/administration/user-group/${" + CommonParameters.ID + "}/", AdministrationUserGroupDescriptionPage.class);
 		
+		// Referential
+		mountPage("/referential/", ReferentialPage.class);
```

Comment:
Not sure about this translation of "référentiel"... Are you?
If you aren't either, maybe we could just avoid the issue and use the "reference" keyword, or "reference-data". Same goes for the page class.